docs/tutorial/appendix/source-JSON-YAML/index.md

```diff
@@ -0,0 +1,274 @@
+---
```

Comment:
I haven't seen an appendix file in the tutorial before...is there precedent for that yet? I do think having a dedicated page for this as part of the tutorial makes sense.